numba/typed/typedlist.py

```diff
@@ -444,7 +444,7 @@ def __repr__(self):
 
 
 # XXX: should we have a better way to classmethod
-@overload_method(TypeRef, 'empty_list')
+@overload_classmethod(ListType, 'empty_list')
```

Comment:
I guess this means `List.empty()` can now written as an alias of this?

Reply:
Yes, we can finally unify the spelling for Dict and List.

Reply:
Great!